src/styles/components/_profile-upload.scss

```diff
@@ -0,0 +1,108 @@
+@import '../abstracts/_abstracts-dir';
+
+.profile-upload {
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
Please use paddings to limit element content.

<img width="777" alt="Screen Shot 2564-03-15 at 15 31 23" src="https://user-images.githubusercontent.com/32289827/111125087-cc60ee00-85a3-11eb-977d-a3fd82ed5fab.png">

Reply:
Thanks for mentioning this, BTW, this margin is the default value from `figure` tag.

Reply:
`box-sizing: border-box` has no effect on margin. Please recheck.